.github/workflows/ios-pr.yml

```
@@ -25,8 +25,14 @@ jobs:
             -   name: Install Fastlane
                 run: bundle config set path '~/.gem' && bundle install
 
-            -   name: install Carthage
-                run: brew install carthage
+            -   name: Cache SPM dependencies
+                uses: actions/cache@v2
+                env:
+                    spm-cache-key: ${{ runner.os }}-spm-${{ hashFiles('iOS/**/Package.resolved') }}
+                    spm-path: ~/**/DerivedData
+                with:
+                    path: ${{ env.spm-path }}
+                    key: ${{ env.spm-cache-key }}
```

Couldn't we use it like this?
```suggestion
                with:
                    path: ${{ runner.os }}-spm-${{ hashFiles('iOS/**/Package.resolved') }}
                    key: ~/**/DerivedData
```

---

I was following the backend example, also I thought that with env variables would be more organized. But we can do this way.